prow/clonerefs/run.go

```diff
@@ -108,19 +110,21 @@ func (o Options) Run() error {
 	return nil
 }
 
-func addHostFingerprints(fingerprints []string) error {
-	path := filepath.Join(os.Getenv("HOME"), ".ssh", "known_hosts")
+func addHostFingerprints(fingerprints []string) (string, error) {
+	// The clonerefs image uses Alpine, where a /tmp folder always exists
+	path := filepath.Join("/tmp", "known_hosts")
```

Comment:
Could we try `os.MkdirAll` with best-effort?

Reply:
I mounted the tmp folder as discussed in the issue, but I also try to create the tmp folder if it doesn't exist.